docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
This won't show up in pants_ignore by default, but we probably want Pants to ignore it. Should probably add to `pants.toml`

Reply:
Good idea.